tools/objc/libtool_check_unique.cc

```diff
@@ -32,43 +37,71 @@ string getBasename(const string &path) {
   return path.substr(idx + 1);
 }
 
-// Returns 0 if there are no duplicate basenames in the object files (both via
-// -filelist as well as shell args), 1 otherwise
-int main(int argc, const char *argv[]) {
+std::vector<string> readFile(const string path) {
+  std::vector<string> lines;
```

Comment:
nit: there are as many uses of vector as of unordered_set. for consistency, maybe add a using statement?